website/api/controllers/entrance/signup.js

```diff
@@ -123,14 +128,15 @@ the account verification message.)`,
         'Authorization':sails.config.custom.cloudProvisionerSecret
       }
     )
-    .timeout(5000)
+    .timeout(10000)// FUTURE: set this timeout to be 5000ms
     .intercept(['requestFailed', 'non200Response'], (err)=>{
       // If we received a non-200 response from the cloud provisioner API, we'll throw a 500 error.
       return new Error('When attempting to provision a new user who just signed up ('+emailAddress+'), the cloud provisioner gave a non 200 response. The incomplete user record has not been saved in the database, and the user will be asked to try signing up again. Raw response received from provisioner: '+err.stack);
     })
-    .intercept({name: 'TimeoutError'}, (err)=>{
-      // If the request timed out, we'll throw a 500 error.
-      return new Error('When attempting to provision a new user who just signed up ('+emailAddress+'), the request to the cloud provisioner took over timed out. The incomplete user record has not been saved in the database, and the user will be asked to try signing up again. Raw error: '+err.stack);
+    .intercept({name: 'TimeoutError'},(err)=>{
+      // If the request timed out, log a warning and return a 'requestToSandboxTimedOut' response.
+      sails.log.warn('When attempting to provision a new user who just signed up ('+emailAddress+'), the request to the cloud provisioner took over timed out. The incomplete user record has not been saved in the database, and the user will be asked to try signing up again. Raw error: '+err.stack);
+      return 'requestToSandboxTimedOut';
```

Comment:
Ok this works better than what I originally suggested.  UX might end up being worse, but this is simpler.  (i.e. better to just not create the user record.)

Also I appreciate the explicitness of this warning.  This will still trigger a P1 alert for now based on how our alerts are configured, but this can be tweaked as a special exception in how the alert works.